userspace/libsinsp/filterchecks.cpp

```diff
@@ -6471,7 +6473,63 @@ uint8_t* sinsp_filter_check_container::extract(sinsp_evt *evt, OUT uint32_t* len
 			m_tstr = "NONE";
 			RETURN_EXTRACT_STRING(m_tstr);
 		}
+		break;
+	case TYPE_CONTAINER_K8S_POD_NAME:
+		if(tinfo->m_container_id.empty())
+		{
+			return NULL;
+		}
+		else
+		{
+			const sinsp_container_info::ptr_t container_info =
+				m_inspector->m_container_manager.get_container(tinfo->m_container_id);
+			if(!container_info)
+			{
+				return NULL;
+			}
+
+			if(container_info->m_labels.empty())
+			{
+				return NULL;
```

Comment:
```suggestion
                                 m_tstr = "NONE"
				RETURN_EXTRACT_STRING(m_tstr);
```

Wondering in which cases we should populate with "N/A".